tests/Unit/Entity/Activity/RegistrationTest.php

```diff
@@ -128,12 +128,8 @@ public function testSetActivity(): void
 
     public function testIsReserve(): void
     {
-        $expected = null;
-        $property = (new ReflectionClass(Registration::class))
-            ->getProperty('reserve_position');
-        $property->setAccessible(true);
-        $property->setValue($this->registration, $expected);
-        $this->assertSame($expected, $this->registration->getReservePosition());
+        $expected = false;
```

Comment:
Isn't it better to test an expected = true case?

Reply:
You're right, thought I had it. Seems I didn't. Now explicitly test for a true and a false

Reply:
As we're using a string value for the reserve_position, this might break later. I'd suggest to replace true with any string value.

Reply:
Think this was a case of responding too early. I'm using `RegistrationRepository::MINORDER()` to create a string value. `$expected = false` is just because I assume `isReserve` will return a false.